### PR TITLE
Add Code of Conduct adapted from last years

### PR DESCRIPTION
**Pull request by dcwalk**
Just pulled last years, wanna run thru details before we add in.

**TODO FOR MERGE**
- [x] discuss at planning meeting (2018-03-24)
- [x] obtain consensus from active organizers
- [x] update with date approved
- [ ] merge!
- [ ] add to website (https://github.com/ournetworks/ournetworks.ca/pull/9)

### CODE REVIEW

**Comment by dcwalk** [comment]
Hey @ana0 @garrying @patcon @jkms @benhylau hey all -- can you review the updated language for Land Acknowledgement and Code of Conduct here? We need to consense on this before going ahead.

File: CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+At events, Our Networks organizers will identify themselves and any incidents can be reported to them directly. Event organizers will be happy to help participants by providing escorts, contacting local law enforcement, or otherwise assisting those experiencing harassment to feel safe for the duration of an event. We value your participation.
+
+If a participant engages in harassing behaviour, the organizers may take any action they deem appropriate, including warning the offender or expulsion from events and online forums.
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact an organizer immediately.
+
+Those who wish to report a violation but don't feel comfortable talking to the organizers online or at an event can email [coc@ournetworks.ca](mailto:coc@ournetworks.ca), which is **currently** monitored by [**Dawn, @dcwalk**](http://dcwalker.ca), and will be monitored during the event on an active basis.
+
+Our [internal guidelines](./CONDUCT.md) for enforcing our Code of Conduct are viewable online.
```
Comment (garrying) [inline CONDUCT.md]: Should this point to `./CONDUCT.md#guidelines`?

File: CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+_These are adapted from the [Toronto Mesh Code of Conduct](https://tomesh.net/code-of-conduct/), which in turn was based on the anti-harassment policy from the [Geek Feminism Wiki](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) and draws inspiration from [No more rock stars: how to stop abuse in tech communities](https://hypatia.ca/2016/06/21/no-more-rock-stars/). The section about expected behaviour is adapted from the [Gamma Space Code of Conduct](https://github.com/GammaSpace/policies/blob/master/code-of-conduct.md)._
+
+---
+
+## Guidelines
+
+Our Networks adopted the [Toronto Mesh Code of Conduct](https://tomesh.net/code-of-conduct/) for our January 14, 2017 event, and consensed to actively use a conference-specific code of conduct on [XXXX, 2018](https://github.com/ournetworks/2018/pull/33). The Code of Conduct language is in our [2018 organizing](./CONDUCT.md) repository and on our website at [ournetworks.ca/code-of-conduct/](https://ournetworks.ca/code-of-conduct).
```
Comment (garrying) [inline CONDUCT.md]: `[2018 organizing](./CONDUCT.md)`: Point to the top-level of the organizing repository?

File: CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
```
Comment (garrying) [inline CONDUCT.md]: Small thought on file naming: Would `CODE_OF_CONDUCT.md` be more clear?

Comment (dcwalk) [reply CONDUCT.md]: 🤷‍♀️ We are going to put it up on the website at /code-of-conduct and link prominently to that, so I think it doesn't matter too much in the repo? 

I've seen it many ways, including as it is now

File: CONDUCT.md (hunk 1 above)
Comment (dcwalk) [reply CONDUCT.md]: 👍

File: CONDUCT.md (hunk 2 above)
Comment (dcwalk) [reply CONDUCT.md]: hmmm, I'm trying to link directly to the coc in each place (website, repo) so I'm inclined to leave as is? Does that make sense?

**Comment by dcwalk** [comment]
Thanks @garrying and @ana0 -- @benhylau and @patcon (@jkms ?) can you please also review?
We should aim to merge in ASAP As some places I want to advertise require a CoC!

File: CONDUCT.md (hunk 3 above)
Comment (garrying) [reply CONDUCT.md]: 👍

File: CONDUCT.md (hunk 2 above)
Comment (garrying) [reply CONDUCT.md]: yeah. makes sense

File: README.md [hunk 4]
```diff
@@ -35,6 +35,16 @@ Our main organizing email is orga@ournetworks.ca. Organizers are:
 - jkms (at large) (gh: [@jkms](https://github.com/jkms))
 - patcon (gh: [@patcon](https://github.com/patcon))
 
+## Code of Conduct
+
+This event has a [Code of Conduct](./CONDUCT) that all organizers and attendees have been asked to follow in order to create a welcoming space to aim to do our best work together.
```
Comment (benhylau) [inline README.md]: CONDUCT.md

**Comment by dcwalk** [comment]
Okay, I've done another round of emailing, it looks like all who are engaged in organizing have given the go ahead! Thanks for feedback, I'll merge this in

**Pull request merged**